### DISCUSSION TITLE
setFilter, first letter only

### DISCUSSION THREAD

**Discussion by orioletech** [TanStack/table#2575] [General]
setFilter works great taking a letter input and filtering all of a specific columns data to only show values containing that letter e.g setFilter("game_title", letter) . However I can't find a solution to make the filter only look at the first letter of each 'game_title' rather than checking the entire word.

I've been working from this documentation  - https://react-table.tanstack.com/docs/examples/filtering

Any help would be amazing!
Many thanks

**Comment by gargroh** [comment]
You need to write ur own custom filter fn for that..

**Reply by orioletech** [reply]
Would you be able to link me an example? I can't find anything that gives me direction on creating a custom setFilter function - would be really appreciated 👍

**Comment by gargroh** [comment]
check filter examples, there should exist one
